syft/pkg/cataloger/binary/classifier_cataloger_test.go

```diff
@@ -1348,6 +1348,17 @@ func Test_Cataloger_PositiveCases(t *testing.T) {
 				Metadata:  metadata("chrome-binary"),
 			},
 		},
+		{
+			logicalFixture: "ffmpeg/7.1.1/darwin-arm64",
```

Comment:
It looks like the original requester of this issue is on ubuntu. Do you think it's worth making a linux-amd64 test fixture? The binary might look quite different, so I'm not sure this would work on the other platform.

Reply:
Sure thing. Would you prefer a linux-amd64 _instead_ or _in_ _addition_ to the existing test?